README.md

```diff
@@ -35,12 +35,20 @@ to help you get it set up.
 
 [![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/magic-modules&tutorial=mmv1/TUTORIAL.md)
 
+## Preparing your environment
```

Comment:
Do we want to set it as a single section? Or should we just simply change the `### Requirements` below to `### Preparing your environment`